From 9394937ee2b194cdc747aa7014c1f35a3ad3ee77 Mon Sep 17 00:00:00 2001
From: Michael Heimpold <mhei@heimpold.de>
Date: Tue, 4 May 2021 15:14:16 +0200
Subject: [PATCH 0692/2201] configs: add CONFIG_QCA7000_UART=m (#4322)

This allows to use the UART interface instead of SPI
of QCA7000 Ethernet over powerline chips.

Signed-off-by: Michael Heimpold <mhei@heimpold.de>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index ca05d73516e2..073877ae2974 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -498,6 +498,7 @@ CONFIG_VETH=m
 CONFIG_NET_VRF=m
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
+CONFIG_QCA7000_UART=m
 CONFIG_WIZNET_W5100=m
 CONFIG_WIZNET_W5100_SPI=m
 CONFIG_MDIO_BITBANG=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 2a20873a30e9..ca0c73831827 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -509,6 +509,7 @@ CONFIG_NET_VRF=m
 CONFIG_BCMGENET=y
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
+CONFIG_QCA7000_UART=m
 CONFIG_WIZNET_W5100=m
 CONFIG_WIZNET_W5100_SPI=m
 CONFIG_MICREL_PHY=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 554f87044043..c71640dd0ce2 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -492,6 +492,7 @@ CONFIG_VETH=m
 CONFIG_NET_VRF=m
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
+CONFIG_QCA7000_UART=m
 CONFIG_WIZNET_W5100=m
 CONFIG_WIZNET_W5100_SPI=m
 CONFIG_MDIO_BITBANG=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 5fb890f90233..23d951e4034f 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -503,6 +503,7 @@ CONFIG_NET_VRF=m
 CONFIG_BCMGENET=y
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
+CONFIG_QCA7000_UART=m
 CONFIG_WIZNET_W5100=m
 CONFIG_WIZNET_W5100_SPI=m
 CONFIG_MICREL_PHY=y
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index ff857505902b..c38726720f27 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -488,6 +488,7 @@ CONFIG_VETH=m
 CONFIG_NET_VRF=m
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
+CONFIG_QCA7000_UART=m
 CONFIG_WIZNET_W5100=m
 CONFIG_WIZNET_W5100_SPI=m
 CONFIG_MDIO_BITBANG=m
-- 
2.17.1

